package.json

```diff
@@ -60,11 +60,33 @@
     "lint-fix": "eslint --fix --ext .js,.vue parliament viewer notifiers wiseService",
     "lint-fix-viewer": "eslint --fix --ext .js,.vue viewer",
     "lint-fix-parliament": "eslint --fix --ext .js,.vue parliament",
-    "lint-fix-wise": "eslint --fix --ext .js,.vue wiseService"
+    "lint-fix-wise": "eslint --fix --ext .js,.vue wiseService",
+    "viewer:addtestuser": "cd viewer && node addUser.js -c ../tests/config.test.ini -n testuser admin admin admin --admin --packetSearch",
+    "viewer:build": "cd viewer && node vueapp/build/build.js",
+    "viewer:bundle": "webpack -d --progress --config viewer/vueapp/build/webpack.dev.conf.js",
+    "viewer:start": "cd viewer && node viewer.js -c ../config.ini",
+    "previewer:start": "npm run viewer:build",
```

Comment:
Is this a hook to run before viewer:start? It doesnt look like we built before this previously